QUEST.tsv

```diff
@@ -10,33 +10,33 @@ QUEST_20150317_000009	Please select a starting zone.
+QUEST_20150317_000018	Instead, I wandered through many far-seeing dreams for an answer. {nl}You were at the end of them… the Savior.
+QUEST_20150317_000019	However, knowing of my clairvoyance, the demons would never rest until they saw me gone. {nl}I was forced to hide away, until the time of your appearance.
+QUEST_20150317_000020	I resolved to divide my being and subsist inside these many messages. {nl}I continued to secretly create them, even as the kingdoms flew by over the millennia.
+QUEST_20150317_000021	Savior, these messages are scattered across axes of time. {nl}Once they're all brought together, I will regain my complete form and help against the demons.
+QUEST_20150317_000022	This divine task may be a heavy burden, but it is something you must bear. {nl}Perhaps, in the muddy midst of your trials and tribulations, something may come about.
 QUEST_20150317_000023	Nevertheless, only you and I can save this world.
-QUEST_20150317_000024	The first Paladin will hold the next one. {nl}I expect you at the highest flower garden.
-QUEST_20150317_000025	The dark aura is indeed unpleasant… {nl}Yet, if this message is truly from the Goddess Laima, then I am simply amazed.
-QUEST_20150317_000026	Well, you should hurry and go find Sir Uska. {nl}I'm sure he can tell you where to go next. {nl}May the Goddess bless your fate…
+QUEST_20150317_000024	I have passed the next revelation to the first Paladin. {nl}I'll be waiting for you at the highest flower garden.
+QUEST_20150317_000025	The dark aura is indeed unpleasant… {nl}But if this message is truly from the Goddess Laima, then I am simply at awe.
+QUEST_20150317_000026	Well, you should hurry and go find Sir Uska. {nl}He will tell you where to go next. {nl}May the Goddess bless your fate…
 QUEST_20150317_000027	Please select the preferred Smart Gen.
 QUEST_20150317_000028	William
-QUEST_20150317_000029	The fee's 100 Silver. {nl}It'll be cheaper than cutting through monster central.
+QUEST_20150317_000029	The fee's 100 silver. {nl}It's cheaper than cutting through monster central.
 QUEST_20150317_000030	Cyneburg
-QUEST_20150317_000031	I charge 100 Silver. {nl}That will cover operations, labor, and what have you.
+QUEST_20150317_000031	That'll be 100 silver coins. {nl}That'll cover operations, labor and other expenses.
 QUEST_20150317_000032	Hey, wait a minute. {nl}You know you need more than that…
 QUEST_20150317_000033	That's no good! {nl}I can't work for nothing, even for a Revelator.
 QUEST_20150317_000034	Wandering Merchant
-QUEST_20150317_000035	Like a dandelion's seeds, I am merely one who follows the wind. {nl}Since we've met in such a curious way, perhaps you'll spare a moment to view my curiosities?
+QUEST_20150317_000035	Like a dandelion's seeds, I am merely one who follows the wind. {nl}Since we've met in such an usual way, perhaps you could spare a moment to view my curiosities?
```

Comment:
It looks like the original is a play on words, and it's intentional to use "curious" twice like that.